src/common/lib/lnurl.ts

```diff
@@ -13,7 +13,10 @@ const fromInternetIdentifier = (address: string) => {
       /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
     )
   ) {
-    const [name, host] = address.split("@");
+    let [name, host] = address.split("@");
+    // remove invisible characters %EF%B8%8F
+    name = name.replace(/[^ -~]+/g, "");
+    host = host.replace(/[^ -~]+/g, "");
```

Comment:
that's a great addition! we have already seen the problem of invisible characters. 
do you think we can even be more restrictive there?

Reply:
this worked for my error `%EF%B8%8F` not sure what other invisible characters could cause such issues. do you know a better regex? I'm terrible with those.